hack/deploy-cnv.sh

```diff
@@ -98,5 +98,10 @@ spec:
   BareMetalPlatform: true
 EOF
 
+if [[ "${KUBEVIRT_RELEASE}" =~ 0.34 ]]; then
+    echo "creating KubeVirt config map"
+    oc create -f "https://storage.googleapis.com/kubevirt-prow/devel/release/kubevirt/kubevirt/v0.34.2/manifests/testing/kubevirt-config.yaml"
```

Comment:
Just noticed this now - shouldn't the ConfigMap be created in `$TARGET_NAMESPACE`? If so, I can fix in #20.